### PR TITLE
Upgrade to Scala 2.11

### PR DESCRIPTION
**Pull request by Hayena**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,14 +4,16 @@ organization := "com.banno"
 
 name := "scala-sitemap"
 
-version := "0.10.1"
+version := "0.10.2"
 
-scalaVersion := "2.10.3"
+scalaVersion := "2.11.4"
```
Comment (adamdecaf) [inline build.sbt]: Could this be a [crossScalaVersions](https://github.com/Banno/jsonz/blob/master/build.sbt#L11) instead? We're not fully converted over to 2.11.x yet.

Comment (Hayena) [inline build.sbt]: Fixed

Comment (adamdecaf) [inline build.sbt]: Awesome, thanks!

**Comment by adamdecaf** [comment]
retest this please

**Comment by adamdecaf** [comment]
ok to test

**Comment by adamdecaf** [comment]
retest this please

**Comment by adamdecaf** [comment]
The tests passed on my machine thanks for the contribution @Hayena!

**Comment by adamdecaf** [comment]
@Hayena you should be able to bump to 0.10.3 for 2.11 support. https://github.com/Banno/scala-sitemap#install

**Pull request merged**